docs/content/users/quickstart.md

```diff
@@ -130,14 +130,16 @@ ddev launch
 
 ## Drupal
 
-=== "Drupal 10"
+=== "Drupal 8+"
```

Comment:
I am uncertain if i would call the Tab Drupal 8+. If you take a look at the latest state on the docs you only have Drupal 10 and Drupal 6/7 https://ddev.readthedocs.io/en/latest/users/quickstart/#__tabbed_3_2 . Drupal 8 and 9 are EOL, the apptype got the version removed and is called "just" drupal. not sure if it is necessary to stick to a version numbering here. i also wonder if it is necessary to keep "drupal 6" named. it is also EOL, it just happens to have the same instructions as drupal 7. but not sure if it is necessary to have versions that are EOL named in a quickstart guide. people who need the functionality running drupal 6 already have an install of drupal 6 and they know what to do. so what not simply call the tabs Drupal and Drupal 7 (legacy) something like that?

Reply:
Changed tab name to "drupal"